sources/api/pluto/src/eks.rs

```diff
@@ -1,33 +1,24 @@
+use aws_sdk_eks::model::KubernetesNetworkConfigResponse;
+use aws_types::region::Region;
 use hyper::http::uri::InvalidUri;
 use hyper::Uri;
 use hyper_proxy::{Proxy, ProxyConnector};
 use hyper_rustls::HttpsConnectorBuilder;
-use rusoto_core::credential::ChainProvider;
-use rusoto_core::region::ParseRegionError;
-use rusoto_core::{Region, RusotoError};
-use rusoto_eks::{DescribeClusterError, Eks, EksClient, KubernetesNetworkConfigResponse};
 use snafu::{OptionExt, ResultExt, Snafu};
 use std::env;
-use std::str::FromStr;
 
 pub(crate) type ClusterNetworkConfig = KubernetesNetworkConfigResponse;
 
 #[derive(Debug, Snafu)]
 pub(super) enum Error {
     #[snafu(display("Error describing cluster: {}", source))]
     DescribeCluster {
-        source: RusotoError<DescribeClusterError>,
+        source: aws_sdk_eks::types::SdkError<aws_sdk_eks::error::DescribeClusterError>,
```

Comment:
nit: I recommend some use declarations to make this path shorter. I think it'll help with readability a bit. This comment applies to other long import paths elsewhere as well.

Reply:
on second thought, I think being explicit here is probably better